stanford/roles/deployment/tasks/main.yml

```diff
@@ -34,9 +34,9 @@
     cidr: "{{ CLUSTER_NAT_SUBNET.cidr }}"
     az: "{{ CLUSTER_NAT_SUBNET.zone }}"
     resource_tags:
-      Cluster: nat
+      Cluster: public
```

Comment:
This is the one set of renaming/refactoring things that I left in place (split to its own commit).

This part is definitely already live already and I don't want to deploy w/out it,
else it will undo the current naming convention.